spec/client/pages_spec.rb

```diff
@@ -18,15 +18,16 @@
 
     shared_examples_for 'list pages' do
       it 'should get pages' do
-        stub_request(:get, pages_url).with(headers: standard_headers, query: {'type' => page_type, 'status' => page_status})
```

Comment:
Does the `query` mocking not work in the latest version? It seemed like a nicer option than constructing the query string ourselves and making assumptions about its ordering.